test/unit/session.spec.js

```diff
@@ -345,39 +339,21 @@ describe('util/session', () => {
         .respondWithSuccess()
         .afterResponse(() => {
           should.not.exist(store.state.request.data.session);
-        });
-    });
-
-    it('logs out immediately for a login within a minute of expiration', () => {
-      const clock = sinon.useFakeTimers();
-      testData.extendedUsers.createPast(1);
-      setData({
-        session: testData.sessions.createNew({
-          expiresAt: '1970-01-01T00:00:59Z'
-        })
-      });
-      store.commit('setSendInitialRequests', false);
-      return mockHttp()
-        .request(() => {
-          logIn(router, store, true);
-          clock.tick(0);
         })
-        .respondWithData(() => testData.extendedUsers.first())
-        .respondWithSuccess()
-        .afterResponses(() => {
-          should.not.exist(store.state.request.data.session);
-        });
+        .finally(cleanup);
     });
 
     it('sets the ?next query parameter', () => {
       const clock = sinon.useFakeTimers();
       testData.extendedUsers.createPast(1);
       return load('/users', {}, false)
-        .restoreSession(true)
+        .respondWithData(() =>
+          testData.sessions.createNew({ expiresAt: '1970-01-01T00:05:00Z' }))
+        .respondWithData(() => testData.extendedUsers.first())
         .respondFor('/users')
         .complete()
         .request(() => {
-          clock.runAll();
+          clock.tick(240000);
```

Comment:
`clock.runAll()` doesn't do what's needed anymore, since we now use `setInterval()` instead of `setTimeout()`. Instead, I specify `expiresAt` explicitly and use `clock.tick()`.